src/core/component.js

```diff
@@ -251,15 +255,28 @@ Component.prototype = {
  */
 module.exports.registerComponent = function (name, definition) {
   var NewComponent;
+  var prevSibling;  // Previous sibling of current script.
   var proto = {};
 
-  var testForUpperCase = new RegExp('[A-Z]+');
+  // Warning if component is statically registered after the scene.
+  if (document.currentScript && document.currentScript !== aframeScript) {
```

Comment:
Pretty much did `require('./scene/scenes').forEach(checkPosition)` so we don't have to do querySelector.